overlord/devicestate/devicestate_test.go

```diff
@@ -351,6 +351,11 @@ func (s *deviceMgrBaseSuite) makeSerialAssertionInState(c *C, brandID, model, se
 	return makeSerialAssertionInState(c, s.brands, s.state, brandID, model, serialN)
 }
 
+func (s *deviceMgrSuite) SetUpTest(c *C) {
```

Comment:
@mardy I suspect this line confused you, but it belongs to deviceMgrSuite, not deiceMgrBaseSuite.

Reply:
Indeed!